homeassistant/components/mqtt/climate.py

```diff
@@ -119,6 +119,31 @@
 CONF_TEMP_MIN = "min_temp"
 CONF_TEMP_STEP = "temp_step"
 
+MQTT_CLIMATE_ATTRIBUTES_BLOCKED = frozenset(
+    {
+        climate.ATTR_AUX_HEAT,
```

Comment:
`precision`& `target_temperature` is missing?

Reply:
I think `precision` is not mapped to an attribute, and the `target_temperature` is mapped to attribute ATTR_TEMPERATURE